core/XH.ts

```diff
@@ -326,6 +330,16 @@ export class XHApi {
         return this.identityService?.username ?? null;
     }
 
+    /**
+     * Logout the current user.
+     * @see HoistAuthModel.logoutAsync
+     */
+    async logoutAsync(): Promise<void> {
+        await this.authModel?.logoutAsync();
+        await this.appModel?.logoutAsync();
```

Comment:
Can we explain in a comment why we are logging out twice here? So that app developers can understand whether or not they need to also log out twice in their own apps.

Reply:
removed the second method, per toms suggestion